bin/dev

```diff
@@ -116,7 +116,7 @@ HEREDOC
 
   def update(args)
     system('cd /usr/local/dev-env && git fetch && git reset --hard origin/master') unless args.first == '--no-pull'
-    exec("ansible-playbook #{root_dir}/ansible/mac.yml -i 127.0.0.1,") if mac?
+    exec("ansible-playbook #{root_dir}/ansible/mac.yml -i 127.0.0.1, -K") if mac?
```

Comment:
What are the `-K` flags here are these additional to the `become_flags`?

Reply:
The `-K` instructs ansible to prompt for the "become password" which in our case is the sudo password. We need it to install/manage the launch daemon. The `become_flags` are actual arguments to `sudo`. The flags I specify are the defaults that ansible uses plus `-k` to force sudo to throw away any existing authorizations and try the password passed to ansible.